modules/multilanguage/src/main/protobuf/multilanguage-protocol.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.ukg.surge.multilanguage.protobuf";
+
+message State {
+  bytes payload = 1;
+}
+
+message Command {
+  string aggregateId = 1;
+  bytes payload = 2;
+}
+
+message Event {
+  string aggregateId = 1;
+  bytes payload = 2;
+}
+
+message ProcessCommandRequest {
+  State state = 1;
+  Command command = 2;
+}
+
+message ProcessCommandReply {
+  bool isSuccess = 1;
+  string rejectionMessage = 2;  // non-empty when isSuccess = false
+  repeated Event events = 3;
+  State newState = 4;
+}
+
+message HandleEventsRequest {
+  State state = 1;
+  repeated Event events = 2;
+}
+
+message HandleEventsResponse {
+  State state = 1;
+}
+
+message ForwardCommandRequest {
+  Command command = 1;
+}
+
+message ForwardCommandReply {
+  bool isSuccess = 1;
+  string rejectionMessage = 2; // non-empty when isSuccess = false
+  State newState = 3;
+  repeated Event loggedEvents = 4;
+}
+
+message GetStateRequest {
+  string aggregateId = 1;
+}
+
+message GetStateReply {
+  string aggregateId = 1;
+  State state = 2;
+}
+
+service MultilanguageGatewayService {
+  rpc ForwardCommand (ForwardCommandRequest) returns (ForwardCommandReply) {}
```

Comment:
Do we want to keep the naming here consistent with `sendCommand` or is there a subtle difference between send & forward?

Reply:
@jeffboutotte The mental I have in my mind is that the gRPC server is "forwarding" the command to Surge and Surge is "sending" the command to the right actor on the right node ...Let me know what you think and I don't mind renaming to sendCommand because it's not a big change, totally up to you .